extension/src/nodes/beehave_tree_node.cpp

```diff
@@ -38,14 +38,59 @@ BeehaveTreeNode::BeehaveTreeNode() {
 BeehaveTreeNode::~BeehaveTreeNode() {
 }
 
+PackedStringArray BeehaveTreeNode::_get_configuration_warnings() const {
+	PackedStringArray warnings = Node::_get_configuration_warnings();
+
+	TypedArray<Node> children = get_children();
+	for (int i = 0; i < children.size(); i++) {
+		Variant x = children[i];
+		Node *child = Object::cast_to<Node>(x);
+
+		if (child && !Object::cast_to<BeehaveTreeNode>(child)) {
+			warnings.append("All children of this node should inherit from BeehaveNode class.");
+			break;
+		}
+	}
+
+	return warnings;
+}
+
 BeehaveTreeNode::TickStatus BeehaveTreeNode::tick(Ref<BeehaveContext> context) {
-	return BeehaveTreeNode::FAILURE;
+	return BeehaveTreeNode::SUCCESS;
```

Comment:
Changed this to success as this is what the 2.X version was doing when I looked. I can swap it back if we want